### DISCUSSION TITLE
Two more questions

### DISCUSSION THREAD

**Discussion by vasigorc** [ScalaConsultants/mesmer#382] [Q&A] [answered]
I've another question regarding `README.md` this time 🙂 

The latter file reads:

> akka.actor.typed.extensions= ["io.scalac.mesmer.extension.AkkaMonitoring"]

Does it mean that mesmer only works with [Akka typed](https://doc.akka.io/docs/akka/2.5.32/typed/index.html)?

**Comment by worekleszczy** [comment, marked as answer]
@vasigorc Hi, thanks you for taking an interest in `mesmer` :) No, `mesmer` targets both `typed` and `classic` actors. This part merely means that we use `akka.typed` underneath to produce some of the metrics, but it shouldn't make any difference for your application - whatever type of actor system you use metrics will be collected for you :)

**Reply by vasigorc** [reply]
Cool, thank you @worekleszczy - resolving this.
